views/question-table.pug

```diff
@@ -1,7 +1,8 @@
 table.table
   thead
     tr
-      th Visible
+      th
+        button(id='vis' type='button' value='true' onClick='toggleButtonVisibility()') Visiblity
```

Comment:
Maybe we can label column (or the button) something like toggle visibility.

Reply:
Or maybe we can leave the column header there and append a really small hyperlinked text that says (show/hide) that will toggle the buttons themselves. DIVA requirement though. Good work otherwise.

Reply:
I see what you're saying, it would be a nice touch.